extensions/cornerstone-dicom-sr/src/viewports/OHIFCornerstoneSRViewport.tsx

```diff
@@ -210,9 +210,16 @@ function OHIFCornerstoneSRViewport(props) {
         // should be passed second since we don't want SR displaySet to
         // override the activeImageDisplaySetData
         displaySets={[activeImageDisplaySetData]}
+        // Note: we shouldn't inherit the viewportOptions
+        // from the parent viewportGrid, since we are rendering the Key Image(s)
+        // for the OHIF SR Viewport, and for the case where the old viewport
+        // had volume data, the SR Viewport would try to render the volume data
+        // and it just fails miserably (since it's not a volume data, and SR
+        // is not hydrated to get the data.handles.points in world coordinates
         viewportOptions={{
           ...viewportOptions,
           toolGroupId: `${SR_TOOLGROUP_BASE_NAME}`,
+          viewportType: 'stack',
```

Comment:
I don't think this is the right choice here.  The issue is caused by how the viewportId gets re-used in an incompatible viewport.  It happens to fix this particular issue, but I don't think it fixes problems generally with other potentially incompatible arguments.

Reply:
We don't have support for volume SR since I saw it requires world coords for the filterAnnotationForVolumeViewport which we don't have right now. So given the current support it should force it to Stack viewport.

Reply:
if you don't put 'stack' here, it tries to load the SRDisplayTool in a volume viewport and we don't have support right now. I prefer removing this after we add support for SCOORD3D